rust/content.md

```diff
@@ -33,20 +33,20 @@ $ docker run -it --rm --name my-running-app my-rust-app
 This creates an image that has all of the rust tooling for the image, which is 1.8gb. If you just want the compiled application:
 
 ```dockerfile
-FROM rust:1.40 as builder
+FROM %%IMAGE%%:1.40 as builder
 WORKDIR /usr/src/myapp
 COPY . .
 RUN cargo install --path .
 
 FROM debian:buster-slim
-RUN apt-get update && apt-get install -y extra-runtime-dependencies
+RUN apt-get update && apt-get install -y <extra-runtime-dependencies>
```

Comment:
ah, this is so helpfull
i actually tried installing `extra-runtime-dependencies` jus to find out such package doeent exist

Reply:
Kinda awkwar when my docker failed to install above, just stumbled upon checking apt and they dont have such package, and TBH I just realized it's a "TERMINOLOGY" to add a package, LMAO

Reply:
@daison12006013 Soooo funny story, ending up here for exactly that reason here! Maybe it's an idea to comment the line out, and put a comment above it to replace `<extra-runtime-dependencies>` with the packages you need.

Reply:
ah, i did the same and thought that it's a real package

Reply:
Me too 😂

Reply:
Is this markdown file have a chance to merge? 😂

Reply:
I was trying to follow the example and it didn't work.  I found this pull request googling and everything instantly made sense. I suggest releasing the documentation improvements and you can always iterate to an even "more" improved set later.

Reply:
Now I feel silly 🥴

Reply:
I thought `extra-runtime-dependencies` will generate magically somewhere... 😵

Reply:
Same here, you should also put examples of those extra dependencies. Or how do you get your dependencies.

Reply:
I also did this mistake, it appears the instructions are still unclear - any change of re-running the expired pipeline and merging this PR? It seems this has got stalled over some minor technicalities?

Reply:
If parts of this patch are contentious, why not split out this piece which seems pretty helpful?

Reply:
should be merged, I fell for it too lmao

Reply:
I also thought this was the name of debian dep! We really should have this fixed.

Reply:
Happened with me also jajaja